bevy_rapier2d/examples/voxels2.rs

```diff
@@ -0,0 +1,105 @@
+use bevy::prelude::*;
+use bevy_rapier2d::prelude::*;
+use nalgebra::{point, Vector2};
```

Comment:
relying on nalgebra types here is not great, but I'm not sure duplicating API on library's side is great either :thinking: